### PR TITLE
Add yea profile support

### PR DESCRIPTION
**Pull request by raubitsj**

### CODE REVIEW

File: src/yea/ytest.py [hunk 1]
```diff
@@ -279,6 +281,21 @@ def _run(self) -> None:
             env["YEA_PARAM_NAMES"] = ",".join(self._permute_groups)
             env["YEA_PARAM_VALUES"] = ",".join(map(str, self._permute_items))
 
+        # pass profile config as env vars to be loaded by yea.setup() in test
+        # NOTE: yea.setup() will not be required in the future (hopefully)
+        profile: List[Union[str, Dict[str, Dict[str, Any]]]] = self._test_cfg.get("profile", [])
+        if profile:
+            prof_vars: str = ",".join(map(lambda p: next(iter(p)) if isinstance(p, dict) else p, profile))
```
Comment (dmitryduev) [inline src/yea/ytest.py:288]: python-kun approves ^_^

File: src/yea/ytest.py [hunk 2]
```diff
@@ -311,8 +328,12 @@ def _run(self) -> None:
                     env[f"YEA_PLUGIN_{penv}_VALUES"] = json.dumps(pvalues)
             env["YEA_PLUGINS"] = ",".join(plugins)
 
+        start_time = time.monotonic()
```
Comment (dmitryduev) [inline src/yea/ytest.py:331]: i hear people say it's best to use time.perf_counter for this purpose

**Review by dmitryduev — APPROVED** [review]
🎉

File: src/yea/ytest.py (hunk 2 above)
Comment (raubitsj) [reply src/yea/ytest.py:331]: ill look into that in the future. thanks..

**Pull request merged**